src/Queue.ts

```diff
@@ -171,7 +171,8 @@ export class Queue {
       exclusive: this.options.exclusive || false,
       arguments: this.arguments
     });
-    await ch.bindQueue(this.options.name, exchange, this.options.topic);
+
+    await Promise.all(this.options.topics.map(topic => ch.bindQueue(this.options.name, exchange, topic)));
```

Comment:
usa "for of" aqui, com await por topicos.

vai deixar o código mais legível e evitar problemas randômicos que possam ocorrer com o driver.

Reply:
Feito, realmente, usar o map é meio complicado quando temos uma instância somente :p